### DISCUSSION TITLE
umail - any working client at the moment?

### DISCUSSION THREAD

**Discussion by bigplik** [micropython/micropython#9103] [Libraries & Drivers] [answered]
Hi, does anyone heard of email client which works with umail at the moment? Or some equivalent solution?
Before I used gmail, but now they turned off access for less secure apps and umail library doesn't work.
I checked yahoo, aol, my.com, gmail, mail - nothing works.

**Comment by davefes** [comment, marked as answer]
I had to change my mail_functions program, which used Gmail SMTP server, to use an application password.
[https://support.google.com/mail/thread/157132285/log-in-to-gmail-using-3rd-party-apps?hl=en](url)
Actually, turned-out a lot easier than I first thought.
